spec/011_music_library_controller_spec.rb

```diff
@@ -37,9 +37,18 @@
 
     it 'asks the user for input at some point' do
       music_libray_controller = MusicLibraryController.new("./spec/fixtures/mp3s")
+      expect(music_libray_controller).to receive(:gets).and_return("exit")
```

Comment:
We should take this opportunity to fix the spelling of "library" throughout this spec :)